service/src/main/resources/api/service_openapi.yaml

```diff
@@ -157,6 +157,32 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  # TODO(zloery): This endpoint is exposed as a workaround to support the CLI's
+  # use of Nextflow. It will likely be removed in the future in favor of other
+  # cloud sync methods.
+  /api/workspaces/v1/{workspaceId}/gcp/enablepet:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+    post:
+      summary: Allow a user to impersonate their pet SA in a given workspace
+      description: |
+        This endpoint is exposed as a workaround to support calling Nextflow in
```

Comment:
Good point about future tools. I only considered Nextflow because that's the one that gets talked about most, but you're right that any other tool calling Pipelines API would also require this endpoint. Updated description